sirepo/package_data/static/html/raydata-run-analysis.html

```diff
@@ -1,5 +1,15 @@
 <div class="container-fluid">
-  <div data-basic-editor-panel="" data-view-name="runAnalysis">
-    <div data-model-field="'scans'" data-model-name="'runAnalysis'"></div>
+  <div data-simple-panel="runAnalysis">
+    <div class="row">
+      <div class="col-md-7">
+        <div data-advanced-editor-pane="" data-view-name="'runAnalysis'" data-field-def="basic" data-want-buttons="true"></div>
+      </div>
+      <div class="col-md-5">
+        <div data-scan-detail="" data-scan="runAnalysis.raydataService.detailScan"></div>
+      </div>
+      <div class="col-md-12">
+        <div data-model-field="'scans'" data-model-name="'runAnalysis'"></div>
+      </div>
+    </div>
```

Comment:
Looks good - two other things:

On the Run Analysis tab, when you mouse over the scan table, it should show a "pointer" cursor to indicate you can click on the item. This shouldn't be the behavior on the Queued tab which doesn't have the Scan Detail panel.

When you enter a new search and select "Save", it should clear the currently displayed scan detail info.

Reply:
19b83fcf06f5a7cf22d5b2519a36962ce1026d56